### ISSUE TITLE
Documentation for analyses not shown in dbt docs for version 0.20.1

### ISSUE THREAD

**Issue by johansjob** [dbt-labs/dbt#3837]
### Describe the bug
Adding a description field to an anaylsis' schema files does not show in the dbt docs when running dbt version 0.20.1

### Steps To Reproduce
- Create a standard dbt project
- Add an analysis
- Document the analysis with a schema file. Make sure to add a description
- With dbt version 0.20.1 run `dbt docs generate` and `dbt docs serve`
- In dbt docs, navigate to the analysis model. The description from the schema file is not shown.

### Expected behavior
Documentation to be shown for analyses

### System information
**Which database are you using dbt with?**
bigquery

**The output of `dbt --version`:**
```
installed version: 0.20.1
   latest version: 0.20.1

Up to date!

Plugins:
  - bigquery: 0.20.1
```

**The operating system you're using:**
Mac OSX

**The output of `python --version`:**
3.8.2

### Additional context
See slack conversation for more info.
https://getdbt.slack.com/archives/C2JRRQDTL/p1630067586034500

**Comment by jtcohen6** [comment]
@johansjob Thanks for the report! I believe this was previously reported in #3764, which we've fixed for v0.20.2 (currently available as a release candidate).

Could you try installing v0.20.2-rc2, generate documentation, and check whether the analysis description shows up?
```
# in a virtual env
pip install dbt-bigquery==0.20.2rc2
```

(I'm going to close this as a dupe in the meantime, but very happy to reopen if we determine it's a new issue)

**Comment by johansjob** [comment]
Aha, that I missed completely. Thought I searched for the issue everywhere.. The 0.20.2rc version works wonderfully, thanks!
